docs/labs/sys_admin_1/lab7-software_management.md

```diff
@@ -761,3 +766,39 @@ Amongst other housekeeping tasks, the final installation step also involves copy
         It is good practice to test a program as a regular user to ensure that regular users can indeed use the program. It is possible that the permissions on the binary are set incorrectly such that only the super-user can use the programs. This of course assumes that you indeed want regular users to be able to use the program.
 
 5. That is it. This lab is complete!
+
+## Exercise 7
+
+### Checking file integrity after package installation
+
+After installing relevant packages, in some cases, I need to determine whether the associated files have been modified to prevent malicious modifications by others.
```

Comment:
```suggestion
After installing relevant packages, in some cases, we need to determine whether the associated files have been modified to prevent malicious modifications by others.
```